entity-service-impl/src/main/java/org/hypertrace/entity/query/service/converter/response/getter/MapValueGetter.java

```diff
@@ -56,16 +61,26 @@ public Value getValue(final JsonNode jsonNode) throws ConversionException {
 
       final String key = entry.getKey();
       final JsonNode node = entry.getValue();
-
-      if (!nestedValueGetter.matches(node)) {
+      boolean valueSet = false;
+      if (nestedValueGetter.matches(node)) {
```

Comment:
Could this be combined with the for-loop below (since nestedValueGetter is already a part of the rootGetters).